### ISSUE TITLE
Jigkick Battery Support

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kuuhakuzx/procfw#12]
```
What steps will reproduce the problem?
1. Insert a battery that has a serial of 0xFFFFFF
2. Turn on psp-1000/2000

What is the expected output? What do you see instead?
PSP booting to VSH and suspending properly. Instead of this the PSP fails to 
boot.

What version of the product are you using? On what operating system?
PSP 02g PRO-B6

Please provide any additional information below.
This is normal OFW behavior and may be beyond the scope of this project, 
however it would be a great addition if possible.

```

Original issue reported on code.google.com by `uberusha...@gmail.com` on 31 May 2011 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
Sadly this is only possible via a proper MMS setup, and as you said beyond the 
scope of this project.

I will close this for now, should we find a way to support this properly this 
issue will be reopened.
```

Original comment by `gentlema...@gmail.com` on 31 May 2011 at 9:14
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
I failed to mention that I already had an MMS, what do we need to get the IPL 
loaded in PRO?
```

Original comment by `uberusha...@gmail.com` on 1 Jun 2011 at 12:29

**Comment by GoogleCodeExporter** [comment]
```
This is now fixed as of revision b3427ba51c3f 
```

Original comment by `uberusha...@gmail.com` on 6 Jun 2011 at 1:12
